homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
By default brightness is only exposed when the device is on.
I don't have isy994, and don't know why here it is exposed always.

@jbcodemonkey added it in https://github.com/home-assistant/home-assistant/commit/a3db0ec231f467391438bf01c8cf466f4d5ec631

@jbcodemonkey Is it indeed required here to always expose brightness?